lib/shelley/bench/Restore.hs

```diff
@@ -600,6 +610,15 @@ dummyAddress
     | otherwise =
         Address $ BS.pack $ 1 : replicate 56 0
 
+-- | Hash the given text, and construct a mnemonic from it.
+dummySeedFromName :: Text -> SomeMnemonic
+dummySeedFromName = SomeMnemonic @24
+    . entropyToMnemonic
+    . unsafeMkEntropy @256
+    . BS.take 256
```

Comment:
surprised here. to my best knowledge Blake2b 256 will create 32-byte bytestring. This line seems unnecessary